miden-rpc/src/server/api.rs

```diff
@@ -0,0 +1,122 @@
+use std::{net::ToSocketAddrs, sync::Arc};
+use tracing::info;
+use anyhow::Result;
+use crate::config::RPCConfig;
+use tonic::{transport::{Channel, Server, Error}, Request, Response, Status, Streaming};
+use prost::Message;
+use miden_proto::{store::{self, api_client}, rpc, digest};
+
+pub struct RpcApi {
+    client: Arc<api_client::ApiClient<Channel>>,
+}
```

Comment:
`client` here is the `store` client, right? If so, I'd call it `store` because we will introduce other components here in the future - e.g., `mempool`, `tx_executor` etc.

Reply:
done